features/build_runner/fastlane_build_runner.rb

```diff
@@ -89,21 +89,23 @@ def run
         logger.info("fastlane run complete")
         logger.debug(build_output.join("\n").to_s)
 
-        return [] # TODO: return artifacts here
+        # return [] # TODO: return artifacts here
       rescue StandardError => ex
         # TODO: Exception handling here
         logger.error(ex)
         logger.error(ex.backtrace)
-        return [] # TODO: return artifacts here (if any)
-        # ensure
+        # return [] # TODO: return artifacts here (if any)
+      ensure
+        # WARN: Delete
         # Either the build was successfull or not, we have to ensure the artifacts for the execution.
-        # artifact_paths = []
-        # artifact_paths << { type: "log", path: "fastlane.log" }
-        # constants_with_path = Fastlane::Actions::SharedValues.constants
-        #                                                      .select { |value| value.to_s.include?("PATH") } # Far from ideal, but meanwhile...
-        #                                                      .select { |value| !Fastlane::Actions.lane_context[value].nil? && !Fastlane::Actions.lane_context[value].empty? }
-        #                                                      .map { |value| { type: value.to_s, path: Fastlane::Actions.lane_context[value] } }
-        # artifact_paths.concat(constants_with_path)
+        artifact_paths = []
+        artifact_paths << { type: "log", path: "fastlane.log" }
+        constants_with_path = Fastlane::Actions::SharedValues.constants
```

Comment:
Let's add some comments about what is happening here and why we need to filter out stuff